src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -333,6 +338,11 @@ export class DynamicFacet extends Component {
     this.logger.info('Show more values');
     this.dynamicFacetQueryController.increaseNumberOfValuesToRequest(additionalNumberOfValues);
     this.triggerNewQuery();
+    this.usageAnalytics.logCustomEvent<IAnalyticsFacetMeta>(
```

Comment:
You could create a private method for logging and use it for show more and show less (only passing the cause as parameter)